releng/feature-announce/upcoming-changes.gomd

```diff
@@ -0,0 +1,24 @@
+{{- /*
+
+    This file is part of the KubeVirt project
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Copyright the KubeVirt Authors.
+
+*/ -}}
+{{- /* gotype: kubevirt.io/project-infra/releng/feature-announce.UpcomingChangesAnnouncementData */ -}}
```

Comment:
Are we able to add a title to this .md: `Upcoming changes for ${version}` ?

Reply:
How do we know what the next release version will be? We could deduce it from the current one, but what if we want to have a new major version?

Reply:
So what would you suggest? Should I add a generic title meta information into the document or is it good enough for now?

Reply:
Probably good enough for now. 
Maybe it makes sense to put this file into a folder, and then we can create a "splash" page (separate PR) for the roadmap that links to this generated doc that also includes the version and the caveat of "These are the currently merged PRs but that does not mean they are 100% guaranteed to be included in the next version etc". We would also then eventually link to the list of design proposals  (with caveats) that we had talked about. 
(It needn't move to a new folder but it would reduce the chance of someone seeing it without the context of the version/caveat). 
What do you think? Also happy for that to be a subsequent iteration so as to move this along.

Reply:
Maybe we can just add the caveat into the markdown template? Something like a paragraph preceding the table, like in this gist: https://gist.github.com/dhiller/50a07158221a42812bdba170184cb4b7